File: app/src/main/java/com/alvinhkh/buseta/kmb/model/KmbRouteBasicInfo.java
package com.alvinhkh.buseta.kmb.model;

import android.support.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class KmbRouteBasicInfo {

  @SerializedName("Racecourse")
  public String racecourse;

  @SerializedName("DestEName")
  public String destinationEn;

  @SerializedName("OriCName")
  public String originTc;

  @SerializedName("ServiceTypeENG")
  public String serviceTypeEn;

  @SerializedName("DestCName")
  public String destinationTc;

  @Nullable
  @SerializedName("BusType")
  public String busType;

  @SerializedName("Airport")
  public String airport;

  @SerializedName("ServiceTypeTC")
  public String serviceTypeTc;

  @SerializedName("Overnight")
  public String overnight;

  @SerializedName("ServiceTypeSC")
  public String serviceTypeSc;

  @SerializedName("OriSCName")
  public String originSc;

  @SerializedName("DestSCName")
  public String destinationSc;

  @SerializedName("Special")
  public String special;

  @SerializedName("OriEName")
  public String originEn;
}

File: app/src/main/java/com/alvinhkh/buseta/kmb/model/KmbRouteBasicInfo.kt
package com.alvinhkh.buseta.kmb.model

import com.google.gson.annotations.SerializedName

data class KmbRouteBasicInfo(
        @SerializedName("Racecourse")
        var racecourse: String = "",

        @SerializedName("DestEName")
        var destinationEn: String = "",

        @SerializedName("OriCName")
        var originTc: String = "",

        @SerializedName("ServiceTypeENG")
        var serviceTypeEn: String = "",

        @SerializedName("DestCName")
        var destinationTc: String = "",

        @SerializedName("BusType")
        var busType: String = "",

        @SerializedName("Airport")
        var airport: String = "",

        @SerializedName("ServiceTypeTC")
        var serviceTypeTc: String = "",

        @SerializedName("Overnight")
        var overnight: String = "",

        @SerializedName("ServiceTypeSC")
        var serviceTypeSc: String = "",

        @SerializedName("OriSCName")
        var originSc: String = "",

        @SerializedName("DestSCName")
        var destinationSc: String = "",

        @SerializedName("Special")
        var special: String = "",

        @SerializedName("OriEName")
        var originEn: String = ""
)